packages/terra-functional-testing/src/config/utils/getConfigurationOptions.js

```diff
@@ -36,7 +37,8 @@ const getConfigurationOptions = (options) => {
       locale,
       port: assetServerPort,
       site,
-      theme,
+      ...(theme ? { theme } : { theme: getDefaultThemeName() }),
+      overrideTheme: theme,
```

Comment:
override theme is only used in the webpack service. I need to know when a theme is specified in the test runner but for all the other services we always need the theme name.